src/BloomExe/Book/UserPrefs.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Newtonsoft.Json;
+using System.IO;
+
+namespace Bloom.Book
+{
+	public class UserPrefs
+	{
+		private bool _loading = true;
+		private string _fileName;
+		private int _mostRecentPage;
+
+		public static UserPrefs Load(string fileName)
+		{
+			if (string.IsNullOrEmpty(fileName))
+				return null;
+
+			var userPrefs = File.Exists(fileName) ? JsonConvert.DeserializeObject<UserPrefs>(File.ReadAllText(fileName)) : new UserPrefs();
+			userPrefs._fileName = fileName;
+			userPrefs._loading = false;
```

Comment:
Aha! I misunderstood so badly I asked the wrong question to begin with. I suppose it SHOULD be obvious that deserialize will call MostRecentPage's setter, but if I didn't figure it out others probably won't. I'd say this calls for a comment at least..."Don't want Save() to do anything when JsonConvert calls property setters"...ideal place to put that would be in a deserialize instance method that sets the flag. Since we don't need that method otherwise...maybe on the variable, or the line in Save() that tests it?
It feels vaguely fragile that the only guarantee that this flag is set when it needs to be is that we only deserialize during Load(). If someone pulled deserialize out into an instance method they might easily miss setting and clearing the flag. But I guess it's OK with a comment.